### ISSUE TITLE
Show window in toolbar

### ISSUE THREAD

**Issue by vertigo220** [gileli121/WindowTop#407]
I just had a minor issue when I tried to change the dark mode setting for one window and accidentally changed it for another, requiring me to change that back then change the intended one. This happened because I had my browser maximized and another, smaller window sitting in the top-right corner, but there was a gap of a few pixels between the top of that window and the top of the screen, so when I hovered over what I thought was the minimize button for the window, I was actually slightly above it and hovering over the minimize button for the browser, which was right behind it. It seems the best way to avoid this issue is to simply display the title of the window in the toolbar, so it's very clear which window is being controlled.

**Comment by gileli121** [comment]
Thank you for the report.  
Does this happen to you often? It does sound quite minor. This could also explain why you want the integration with the minimize button to work better.  

Regarding displaying a title, I'm not sure how to do that. Maybe a tooltip could be added to the arrow, but it might make it feel a bit cluttered.

**Comment by vertigo220** [comment]
No, and it is minor. I just figured the fix would also be minor. I know it's possible to get a window's title and other info, so it seems WindowTop could just do that and display it in the toolbar above or below the icons. Alternately, perhaps when the toolbar is showing, or when hovering over a button in it, it could show a border around the window to identify it. But yes, this is minimal enough that doing nothing would only likely result in an infrequent mistake; I just figure better to make that likelihood even lower.
